packages/api/db/migration/20240620131606_add_permissions_for_animal_identifier_type.js

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+
+const PERMISSION_NAME = 'get:animal_identifier_types';
+
+export const up = async function (knex) {
+  // Synchronize the permissions_permission_id_seq sequence with the latest permission_id value.
+  await knex.raw(`
```

Comment:
This is to let the db auto generate the id for new rows because it's currently out of sync and you have to manually look it up and add it in.